src/schema/types/group.graphql

```diff
@@ -0,0 +1,19 @@
+type Group {
+  id: ID!
+  product: Product!
+  type: TypeEnum!
+  minParticipants: Int!
+  maxParticipants: Int
+  name: String!
+  discount: String
+  expiresAt: Date!
+}
+
+enum TypeEnum {
```

Comment:
Let's rename this to "GroupType". I think it's clearer this way.

Reply:
I thought about it, but i think it's redundant, it is already inside the group shema.

Reply:
Ok, it's done